app/src/main/java/comp3350/mealbuddy/objects/User.java

```diff
@@ -0,0 +1,22 @@
+package comp3350.mealbuddy.objects;
+
+public class User {
+
+
+
+    private String name;
+    private int height;
+
+    public enum Sex{
+        MALE, FEMALE;
+    }
+
+    public enum ActivityLevel{
+        LOW, MEDIUM, HIGH;
+    }
+
+    public User(String name, int height) {
+        this.name = name;
+        this.height = height;
+    }
```

Comment:
there needs to be a lot more here, 

You create a gender but you don't have it in the constructor.

```suggestion
    public enum Sex
    {
        MALE, FEMALE;
    }
     public enum ActivityLevel 
     {
        LOW, MEDIUM, HIGH;
    }
    private String name;
    private double height; //converted to cm
    private double weight; //converted to lb
    private Sex sex;

    //minimum constructor
    public User(String name, Sex sex) {
        this.name = name;
        this.sex = sex;
    }
    //set up getters and setters
    
```